WinUIGallery/DataModel/ControlInfoData.json

```diff
@@ -178,6 +178,128 @@
         }
       ]
     },
+    {
+      "UniqueId": "XamlFundamentalsItem",
```

Comment:
There's no need to add this to `ControlInfoData.json` right? The design guidance, a11y and XAML fundamentals sections are hardcoded on the navview page itself?

Reply:
I simply followed the same approach used in the "Design Guidance" and "Accessibility" sections.

![image](https://github.com/user-attachments/assets/a13a08fb-5bab-4100-84fe-74cba168f494)
---
![image](https://github.com/user-attachments/assets/5150b104-ed89-4c0b-9ddc-d09ca0db95cb)

Reply:
Ah, my bad. Thought these were not part of the JSON. We probably want to add a description as well? We can do that in a separate PR if needed

Reply:
I agree, I will open a PR after finishing this